src/applications/static-pages/download-1095b/sass/download-1095b.scss

```diff
@@ -0,0 +1,8 @@
+@import "~@department-of-veterans-affairs/css-library/dist/tokens/scss/variables";
+
+.download-links {
+    border-top: 1px solid var(--vads-color-base-lighter);
+    margin-bottom: 12px;
+    margin-top: 12px;
+    padding-top: 24px;
+}
```

Comment:
It's a good practice to use `rem` for units as this allows the user's scaling preferences to apply. In some cases, like increasing font size, the `px` values in our css would remain fixed which results in disproportionate spacing/size.

The base font size is 16px, so 16px = 1rem. To calculate these other values as rems we can divide wthem by 16 also.
12/16 = .75rem
24/16 = 1.5rem